pkg/imrpc/proxy.proto

```diff
@@ -119,10 +120,11 @@ message EngineSnapshotPurgeStatusProxyResponse {
 message EngineSnapshotCloneRequest {
 	ProxyEngineRequest proxy_engine_request = 1;
 
-	string from_controller = 2;
```

Comment:
IIUC, dIrectly changing this input means that we need to bump up `InstanceManagerProxyAPIMinVersion` as well.

Reply:
I don't think so. We are not changing the way the field is used at all, nor are we changing its tag. Clients should put the address of the snapshot's source engine on the wire with tag 2 and servers should pull it off the wire with tag 2 regardless of how it is named. (https://earthly.dev/blog/backward-and-forward-compatibility/#adding-new-fields)

However, I only made this change to try to follow through on the request in https://github.com/longhorn/longhorn-instance-manager/pull/229#discussion_r1247852573. I can revert to using `from_controller` if preferred. (IMO this would be better than bumping `InstanceManagerProxyAPIMinVersion`).

Reply:
Ah. The beauty of protobuf saves us